actions/image_partition_action.go

```diff
@@ -42,6 +42,7 @@ Yaml syntax for partitions:
 	   start: offset
 	   end: offset
 	   flags: list of flags
+	   no_fsck: bool
```

Comment:
Why is this a negation? seems better to just call it fsck or enable-fsck or somesuch and default to true. negative options are confusing :)

Reply:
fsck is enabled by default, so need to switch it off explicitly.
will add `fsck: bool` then